### PR TITLE
docs: add Docs & Tools WG meeting notes for 2019-04-19

### PR DESCRIPTION
**Pull request by malept**

### CODE REVIEW

File: wg-docs-tools/meeting-notes/2019-04-19.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# Docs & Tools WG meeting 2019-04-19
+
+## Attendees
+
+* @felix
```
Comment (HashimotoYT) [inline wg-docs-tools/meeting-notes/2019-04-19.md]:
```suggestion
* @felixrieseberg
```

Comment (malept) [reply wg-docs-tools/meeting-notes/2019-04-19.md]: Hm, are we doing GitHub handles or Slack handles? I seem to have mixed the two up.

Comment (felixrieseberg) [reply wg-docs-tools/meeting-notes/2019-04-19.md]: I should probably just have the same across both, but as long as I’m the only Felix in the maintainers org I’m fine with either :P

Comment (malept) [reply wg-docs-tools/meeting-notes/2019-04-19.md]: I just made it Slack handles this time.

**Pull request merged**